Sentry.xcodeproj/xcshareddata/xcbaselines/63AA76641EB8CB2F00D153DE.xcbaseline/4487886E-B920-4AB1-99BD-1A9C5128C731.plist

```diff
@@ -11,7 +11,7 @@
 				<key>com.apple.XCTPerformanceMetric_WallClockTime</key>
 				<dict>
 					<key>baselineAverage</key>
-					<real>0.015</real>
+					<real>0.016</real>
```

Comment:
L: is this part of this PR?

Reply:
Good question 👀.  Yep, this is the baseline for the performance test. As we have now one serialization more it is a little bit slower.